wai-extra/Network/Wai/Parse.hs

```diff
@@ -19,6 +19,8 @@ module Network.Wai.Parse
     , File
     , FileInfo (..)
     , parseContentType
+    , ParseRequestBodyOptions (..)
```

Comment:
By exposing the data constructor here, we make it impossible to add new fields without breaking backwards compatibility. I'd rather go for a setter-based approach, which is generally more extensible.